docs/source/programming/GIN-repositories.md

```diff
@@ -0,0 +1,338 @@
+# Create a GIN private repository
+
+[GIN](https://gin.g-node.org/G-Node/Info/wiki) (hosted by the German Neuroinformatics Node) is a free and open data management system designed for neuroscientific data.
+
+ It is web-accessible, based on `git` and `git-annex`, and allows you to keep your data in sync, backed up and easily accessible.
+
+Below we explain the main user workflows in GIN, focusing on creating a private repository of your data.
+
+:::{note}
+All GIN repos are private by default.
+:::
+
+## Preparatory steps - do only once
+
+These steps apply to any of the workflows below, but we need to do them only the first time we use GIN in our machine.
+
+1. Create [a GIN account](https://gin.g-node.org/user/sign_up)
+2. [Download GIN CLI](https://gin.g-node.org/G-Node/Info/wiki/GIN+CLI+Setup#setup-gin-client) and set it up by running:
+   ```
+   gin login
+   ```
+3. Confirm that everything is working properly by typing:
+   ```
+   gin --version
+   ```
+
+## Create a GIN private repository from the command line (CLI)
+
+1. **Log in to the GIN server**
+
+   Before running any `gin` commands, make sure you are logged in to your account by running:
+
+   ```
+   gin login
+   ```
+
+   - You will be prompted for your GIN username and password.
+   - To list the repositories available to your account: `gin repos --all`
+
+:::{tip}
+You may need `sudo` permissions for some of the following `gin` commands. If so, remember to prepend all commands with `sudo`.
+:::
+
+2. **Initialise a GIN repository**
```

Comment:
May be a good opportunity to use [tabs](https://sphinx-design.readthedocs.io/en/latest/tabs.html) for the different options (it could improve readability).